### ISSUE TITLE
SignaturePad WASM Error 

### ISSUE THREAD

**Issue by PatTheLad** [CodeBeamOrg/CodeBeam.MudExtensions#422]
- **Framework:** Blazor WASM Standalone on .NET 8
- **Browser:** Edge
- **IDE:** VS Code
- **Host:** Windows 10 Server with IIS
- **Nuget Package Version:** 6.9.2
- **Component Used:** SignaturePad

**Description:**
In debug mode everything works fine,
but as soon as I publish it,
I get the following error in the browser:
```
crit: Microsoft.AspNetCore.Components.WebAssembly.Rendering.WebAssemblyRenderer[100]
      Unhandled exception rendering component: ConstructorContainsNullParameterNames, <>f__AnonymousType0`4[System.Decimal,System.String,System.String,System.String] SerializationNotSupportedParentType, System.Object Path: $.
System.NotSupportedException: ConstructorContainsNullParameterNames, <>f__AnonymousType0`4[System.Decimal,System.String,System.String,System.String] SerializationNotSupportedParentType, System.Object Path: $.
 ---> System.NotSupportedException: ConstructorContainsNullParameterNames, <>f__AnonymousType0`4[System.Decimal,System.String,System.String,System.String]
   at System.Text.Json.ThrowHelper.ThrowNotSupportedException_ConstructorContainsNullParameterNames(Type )
   at System.Text.Json.Serialization.Metadata.DefaultJsonTypeInfoResolver.PopulateParameterInfoValues(JsonTypeInfo )
   at System.Text.Json.Serialization.Metadata.DefaultJsonTypeInfoResolver.CreateTypeInfoCore(Type , JsonConverter , JsonSerializerOptions )
   at System.Text.Json.Serialization.Metadata.DefaultJsonTypeInfoResolver.CreateJsonTypeInfo(Type , JsonSerializerOptions )
   at System.Text.Json.Serialization.Metadata.DefaultJsonTypeInfoResolver.GetTypeInfo(Type , JsonSerializerOptions )
   at System.Text.Json.JsonSerializerOptions.GetTypeInfoNoCaching(Type )
   at System.Text.Json.JsonSerializerOptions.CachingContext.CreateCacheEntry(Type type, CachingContext context)
--- End of stack trace from previous location ---
   at System.Text.Json.JsonSerializerOptions.CachingContext.CacheEntry.GetResult()
   at System.Text.Json.JsonSerializerOptions.CachingContext.GetOrAddTypeInfo(Type , Boolean )
   at System.Text.Json.JsonSerializerOptions.GetTypeInfoInternal(Type , Boolean , Nullable`1 , Boolean , Boolean )
   at System.Text.Json.WriteStackFrame.InitializePolymorphicReEntry(Type , JsonSerializerOptions )
   at System.Text.Json.Serialization.JsonConverter.ResolvePolymorphicConverter(Object , JsonTypeInfo , JsonSerializerOptions , WriteStack& )
   at System.Text.Json.Serialization.JsonConverter`1[[System.Object, System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].TryWrite(Utf8JsonWriter , Object& , JsonSerializerOptions , WriteStack& )
   at System.Text.Json.Serialization.Converters.ArrayConverter`2[[System.Object[], System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e],[System.Object, System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].OnWriteResume(Utf8JsonWriter , Object[] , JsonSerializerOptions , WriteStack& )
   at System.Text.Json.Serialization.JsonCollectionConverter`2[[System.Object[], System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e],[System.Object, System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].OnTryWrite(Utf8JsonWriter , Object[] , JsonSerializerOptions , WriteStack& )
   at System.Text.Json.Serialization.JsonConverter`1[[System.Object[], System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].TryWrite(Utf8JsonWriter , Object[]& , JsonSerializerOptions , WriteStack& )
   at System.Text.Json.Serialization.JsonConverter`1[[System.Object[], System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].WriteCore(Utf8JsonWriter , Object[]& , JsonSerializerOptions , WriteStack& )
   Exception_EndOfInnerExceptionStack
   at System.Text.Json.ThrowHelper.ThrowNotSupportedException(WriteStack& , NotSupportedException )
   at System.Text.Json.Serialization.JsonConverter`1[[System.Object[], System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].WriteCore(Utf8JsonWriter , Object[]& , JsonSerializerOptions , WriteStack& )
   at System.Text.Json.Serialization.Metadata.JsonTypeInfo`1[[System.Object[], System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].Serialize(Utf8JsonWriter , Object[]& , Object )
   at System.Text.Json.JsonSerializer.WriteString[Object[]](Object[]& , JsonTypeInfo`1 )
   at System.Text.Json.JsonSerializer.Serialize[Object[]](Object[] , JsonSerializerOptions )
   at Microsoft.JSInterop.JSRuntime.InvokeAsync[IJSVoidResult](Int64 , String , CancellationToken , Object[] )
   at Microsoft.JSInterop.JSRuntime.<InvokeAsync>d__16`1[[Microsoft.JSInterop.Infrastructure.IJSVoidResult, Microsoft.JSInterop, Version=8.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60]].MoveNext()
   at Microsoft.JSInterop.JSRuntimeExtensions.InvokeVoidAsync(IJSRuntime , String , Object[] )
   at MudExtensions.MudSignaturePad.OnAfterRenderAsync(Boolean firstRender)
```

**Code Snippet:**
```
<MudSignaturePad Elevation="4" ShowLineWidth=false ShowStrokeStyle=false ShowDownload=false ShowLineCapStyle=false ShowLineJoinStyle=false LocalizedStrings="localizedStrings" Value="signature" ValueChanged="@(x => {bericht.UnterschriftKunde = Encoding.UTF8.GetString(x);})">
</MudSignaturePad>
```

### Thanks in advance!

**Comment by mckaragoz** [comment]
It fixed with #394. It should work on latest RC release.

**Comment by PatTheLad** [comment]
> It fixed with #394. It should work on latest RC release.

Well I am using the latest RC, but sadly it doesnt work. :(

**Comment by mckaragoz** [comment]
@Zonorai do you have idea?

**Comment by mckaragoz** [comment]
> Well I am using the latest RC, but sadly it doesnt work. :(

Are you sure using latest RC? In issue you write 6.9.2 version. The fix came after 7.0.0-preview.3

**Comment by PatTheLad** [comment]
> > Well I am using the latest RC, but sadly it doesnt work. :(
> 
> Are you sure using latest RC? In issue you write 6.9.2 version. The fix came after 7.0.0-preview.3

Oh yeah nvm., didnt consider using a preview version.
Going to test that now!

**Comment by PatTheLad** [comment]
> > Well I am using the latest RC, but sadly it doesnt work. :(
> 
> Are you sure using latest RC? In issue you write 6.9.2 version. The fix came after 7.0.0-preview.3

Fixed the problem thanks!

**Comment by mckaragoz** [comment]
> Fixed the problem thanks!

Great. Don't forget to look at migration guides: #371 and https://github.com/MudBlazor/MudBlazor/issues/8447
